src/lib/db/project-store.js

```diff
@@ -51,11 +51,10 @@ class ProjectStore {
     }
 
     async create(project) {
-        await this.db(TABLE)
+        const [id] = await this.db(TABLE)
```

Comment:
Is catching no longer relevant here?

Reply:
Not really. This was actually a regression I haven't spotted before touching this code again. Swallowing these throws is kind of bad for the user.